docs/src/joint/api/dia/attributes/magnetSelector.html

```diff
@@ -0,0 +1,17 @@
+<p>The node denoted by the attribute value (a selector) will act as a proxy for the magnet (when the subelement becomes a source/target of a link). This will affect the following areas:
```

Comment:
```suggestion
<p>Designate another subelement of the cell as a proxy target for the cell's magnet, whenever this subelement (the one on which this attribute is defined) becomes the source/target of a link. (Usually, this attribute needs to be defined on the `'root'` subelement of the cell, since that is the default target for the cell's magnet.) Expects a selector.</p>

<p>A change in the cell's magnet affects the following characteristics of the cell view:
```